Лабораторная работа 3

GhosT-FlexAgen opened the pull request.

GhosT-FlexAgen committed:
Add files via upload

dmitrylaabe commented on Лабораторная работа 3/task_1.py:
```diff
@@ -0,0 +1,14 @@
+def get_product_index(products_list, target_products):
+    if target_products not in products_list:
+        return None
+    return products_list.index(target_products)
```
Ф-ю index использовать в учебном задании не совсем правильно, так как по Заданию Вам нужно ее написать. Используйте цикл.

GhosT-FlexAgen committed:
Исправление недочётов Задания 1

Поиск индекса элемента переписан на цикл (без использования встроенной ф-и index).
